docs/UsersGuide.asciidoc

```diff
@@ -224,6 +224,11 @@ mean existing jobs for earlier builds with the same DISTRI and VERSION are
 no longer interesting, but you still want to be able to re-submit jobs for a
 build and have existing jobs for the exact same build obsoleted.
 
+_GROUP:: Job templates *not* matching the given group name are ignored. Does *not*
```

Comment:
Should be easy to make this a comma split list of regexes, WDYT?

Reply:
Sounds like a different user story.